pkg/csi_driver/node.go

```diff
@@ -133,11 +168,6 @@ func (s *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublish
 	}
 
 	// Check if the sidecar container was injected into the Pod
```

Comment:
Should we move and/or delete this comment since you moved 
```
	pod, err := s.k8sClients.GetPod(vc[VolumeContextKeyPodNamespace], vc[VolumeContextKeyPodName])
	if err != nil {
		return nil, status.Errorf(codes.NotFound, "failed to get pod: %v", err)
	}
	gcsFuseSidecarImage := gcsFuseSidecarContainerImage(pod)
```